uploadNeuroDB/bin/mincpik

```diff
@@ -1,4 +1,156 @@
 #! /usr/bin/env perl
+
+=pod
+
+=head1 NAME
+
+mincpik - generate an image from a set of MINC volumes
+
+=head1 SYNOPSIS
+
+mincpik [options] minc_file [outfile]
+
+=head1 DESCRIPTION
+
+This script generates an image from the MINC volumes found in the file passed on
+the command line. Various scripts and external tools are used to accomplish this:
+C<mincinfo>, C<mincreshape>, C<mincextract>, C<convert>, C<rawtominc>, C<montage >
+and the open-source ImageMagick software (http://www.imagemagick.org). The list of 
+options that affect the behaviour of C<mincpik> is as follows:
+
+=over 4
+
+=item *
+B<-verbose> : be verbose
+
+=item *
+B<-clobber> : if it exists, overwrite the output file. An error is produced if F<outfile> exists
+and this option is not used.
+
+=item *
+B<-fake> : do not actually run the command but instead print on STDOUT all the calls to the
+ scripts and external tools (see above) issued to generate the image
+ 
+=item *
+B<-slice index> : C<index> (starts at 0) of the slice to use to generate the image. Defaults to
+C<int(n/2)> (where C<n> is the total number of slices) if not specified.
+
+=item *
+B<-scale scale> : scaling factor to use when generating the image. Defaults to 2. 
+Note that this option is ignored if C<-width> is used (see below)
+
+=item *
+B<-width width> : autoscale the image so it has a fixed width (in number of pixels). 
+If this option is used, then the scaling factor (-scale option) is ignored.
+
+=item *
+B<-depth 8|16> : bitdepth for resulting image (8 or 16). This option should be used 
+on big-endian machines only.
+
+=item *
+B<-title> : whether the image should be generated with a title or not
+
+=item *
+B<-title_text title> : title text for the generated image
+
+=item *
+B<-title_size size> : font size (in pt.) to use when generating the image title
+
+=item *
+B<-anot_bar text> : use the supplied text argument as the annotation bar for the image.
+Note that the image height will be used to determine the bar size.
+
+=item *
+B<-lookup -hotmetal|-gray|-grey|-spectral>: use the specified lookup table to compute RGB
+values when generating the images (see the manpage for C<minclookup>).
+
+=item *
+B<-range min max> : specifies  the  range  of voxel values to consider, in their
+integer representation. Default is the full range for the type and sign. 
+This option is ignored for floating point values. See manpage for C<mincreshape>.
+
+=item *
+B<-image_range min max> : Normalize the image to a given C<min> and C<max> real value (not
+voxel value). Cannot be used with C<-auto_range>. See manpage for C<mincreshape>.
+
+=item *
+B<-auto_range> : Automatically find the range used to normalize the image. This is
+done by finding the 5% to 95% PcT image range using C<mincstats>. This option
```

Comment:
what is PcT?

Reply:
Postponing documentation of this. it for now. Will have to ask Bert about this one...